src/prefect_server/api/runs.py

```diff
@@ -155,6 +174,18 @@ async def _create_flow_run(
     elif flow.archived:
         raise ValueError(f"Flow {flow.id} is archived.")
 
+    # set labels
+    run_labels = []
```

Comment:
This is never hit, the `else` branch handles it.

Reply:
I usually do this for readability so it's clear I expect this to be a list, but I can remove